test/test-pages/common/sdk-controller.js

```diff
@@ -159,9 +159,8 @@ function makeOutboundCall () {
     return;
   }
 
-  let body = { phoneNumber: numberToCall };
-  conversationsApi.postConversationsCalls(body)
-    .catch(err => console.log(err));
+  let body = { sessionType: 'softphone', phoneNumber: numberToCall };
+  webrtcSdk.sessionManager.startSession(body);
```

Comment:
I think callers shouldn't have to call down to the `sessionManager` with the correct `sessionType` to start phone calls. I think it should mimic the [sdk.startVideoConference](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/blob/master/src/client.ts#L274) call directly on the Sdk class. Maybe something like: `sdk.startSoftphoneCall` or something like that. Then that method can call through to `sdk.sessionManager.startSession)`

Reply:
This is a great suggestion because I admittedly didn't look hard enough to see why there are like four `startSession` functions but now I understand.